### PR TITLE
fix issue #7

### PR DESCRIPTION
**Pull request by phoenixking25**
fixed possible error conditions if info is missing.

### CODE REVIEW

File: routes.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 session = database_handler.connect_to_database()
 
 
-@obscura.route('/player/', methods = ['GET'])
+# @obscura.route('/player/', methods = ['GET'])
```
Comment (AnshulMalik) [inline routes.py]: What's this comment for?

File: routes.py [hunk 2]
```diff
@@ -48,36 +49,41 @@ def signup():
 
 
 @obscura.route('/level/', methods = ['GET', 'POST'])
-def getAlias():
+def getalias():
```
Comment (AnshulMalik) [inline routes.py]: Camel Case was looking good

File: routes.py [hunk 3]
```diff
@@ -48,36 +49,41 @@ def signup():
...
+        else:
+            return {'status': 'Not Accessible for the player'}, 403
     else:
-        return {'status': 'Not Accessible for the player'}, 403
+        return {'status': 'invalid access'}, 401
 
-@obscura.route('/getLevel/<alias>', methods = ['GET', 'POST'])
-def getLevel(alias):
+@obscura.route('/getlevel/<alias>', methods = ['GET', 'POST'])
+def getlevel(alias):
```
Comment (AnshulMalik) [inline routes.py]: Make it camel case

File: routes.py [hunk 4]
```diff
@@ -48,36 +49,41 @@ def signup():
...
+            return {'status': 'success', 'alias': level.name}, 200
+        else:
+            return {'status': 'Not Accessible for the player'}, 403
     else:
-        return {'status': 'Not Accessible for the player'}, 403
+        return {'status': 'invalid access'}, 401
 
-@obscura.route('/getLevel/<alias>', methods = ['GET', 'POST'])
-def getLevel(alias):
+@obscura.route('/getlevel/<alias>', methods = ['GET', 'POST'])
```
Comment (AnshulMalik) [inline routes.py]: Should be a only `GET` request on `/level/<alias>` endpoint, not `POST`

**Review by AnshulMalik — CHANGES_REQUESTED** [review]

File: routes.py (hunk 1 above)
Comment (phoenixking25) [reply routes.py]: i have changed url in new commit that i am going to commit so i commented the old ones
to make changes in frontend

Comment (AnshulMalik) [reply routes.py]: Going to? You already committed, and we can see what you changed in git diff, so we can remove the comment

File: routes.py [hunk 5]
```diff
@@ -66,8 +66,9 @@ def getalias():
     else:
         return {'status': 'invalid access'}, 401
 
-@obscura.route('/getlevel/<alias>', methods = ['GET', 'POST'])
-def getlevel(alias):
+
+@obscura.route('/level/<alias>', methods = ['GET', 'POST'])
```
Comment (AnshulMalik) [inline routes.py]: POST method is not required here.

Comment (phoenixking25) [reply routes.py]: its for posting the ans

Comment (AnshulMalik) [reply routes.py]: Then GET is not required

Comment (phoenixking25) [reply routes.py]: get is for getting the level on the basis of Alias parameter :P

Comment (AnshulMalik) [reply routes.py]: Cool

Comment (phoenixking25) [reply routes.py]: Yo

**Pull request merged**